lib/main_common.pm

```diff
@@ -255,6 +256,17 @@ sub load_slepos_tests {
     }
 }
 
+sub load_docker_tests {
+    loadtest "console/docker";
+    loadtest "console/runc";
+    # No package 'docker-compose' in SLE
+    if (check_var('DISTRI', 'opensuse')) {
```

Comment:
Sorry, I should have said in my last review that `check_var('DISTRI', 'opensuse')` could also be replaced by `is_leap || is_tumbleweed` (I think your code work on both openSUSE distribution).

Reply:
Thanks, addressed.